account-kit/react/src/hooks/useSignTypedData.ts

```diff
@@ -39,7 +39,9 @@ export type UseSignTypedDataResult = {
 };
 
 /**
- * Hook for signing typed data, supporting both connected accounts and clients.
+ * Similar to `useSignMessage`, [hook](https://github.com/alchemyplatform/aa-sdk/blob/main/account-kit/react/src/hooks/useSignTypedData.ts) for signing typed data, supporting both connected accounts and clients in EIP 712 format.
+ *
+ * Uses `eth_signtypedmessage` to sign structured, typed data. Accepts typed, complex data structures as input. Like `useSignMessage`, this hook also handles deployed (1271) and undeployed accounts (6492).
```

Comment:
should this be `eth_signTypedData` instead of `eth_signtypedmessage`?

https://github.com/ethereum/EIPs/blob/master/EIPS/eip-712.md